packages/alfa-rules/test/sia-r83/rule.spec.tsx

```diff
@@ -459,6 +459,64 @@ test(`evaluate() passes a text node with fixed height and antoher property
   ]);
 });
 
+/* here */
```

Comment:
**question:** are these comments some remnants that should be removed? (also on line 491)